tests/testthat/test-classSelection.R

```diff
@@ -0,0 +1,65 @@
+context("Does binary class selection work?")
+library(caret)
+library(caretEnsemble)
+
+data(models.class)
+data(X.class)
+data(Y.class)
+
+#############################################################################
+context("Do classifier predictions use the correct outcome classes?")
```

Comment:
I would run this test twice, once where the postive level is "yes" and once where the positive level is "no", and check that no re-arrangement happens, regardless of the alphabetical sorting of the target.